build-system/tasks/bundle-size.js

```diff
@@ -100,10 +109,7 @@ async function storeBundleSize() {
       json: true,
       body: {
         token: process.env.BUNDLE_SIZE_TOKEN,
-        // TODO(#20843, danielrozenberg): add extensions and other runtimes.
-        bundleSizes: {
```

Comment:
Nice foresight designing this api/structure

Reply:
Just following the steps that I detailed in https://github.com/ampproject/amp-github-apps/issues/271 :)